docs/get-started/get-started.md

```diff
@@ -314,7 +314,7 @@ GravatarQuickEditor.logout(Email("{USER_EMAIL}"))
 #### Exclude Data Store files from Android backup (optional, but recommended)
 
 Data Store files are subject to Android backups. Encrypted files from the backup won't work when restored on a different device so we have to exclude those files.
-If your app has backup rules configured, those that are provided in the SDK won't be used so you have to copy them to your files.
+It is encouraged to add the below rules to your own files.
```

Comment:
Third-party developers must copy/paste our rules into their files, right? Do you think we should make it more specific?

Reply:
Yes, correct. Do you want to use the `copy/paste` wording instead of `add`?

Reply:
It's not a strong opinion. I've mentioned that because, at first glance, you can understand that adding the line in the manifest is enough, but maybe it's just me making it clear enough. So, if you think it is okay, feel free to merge it.

Reply:
Updated!